### PR TITLE
fix: make an installation directory just before installing

### LINKED ISSUE #388: インストール先フォルダを指定する前に既定のインストール先にフォルダとファイルを作りに行く
## バグの説明

インストール先フォルダを指定する前に既定のインストール先にフォルダとファイルを作りに行く

インストール先を変えたいときにゴミが発生してしまう

## 再現手順

動作を再現する手順:

1. インストーラーを起動
2. `C:\Users\<user name>\aviutl`が作成されその下に`apm.json`が作成される

## 期待していた動作

実際にインストールするまでフォルダとファイルが作られない

## スクリーンショット

image

## デスクトップ（以下の情報を入力してください）:

- OS: Windows 10 Pro 21H2
- Version: [v2.0.0](https://github.com/hal-shu-sato/apm/releases/tag/v2.0.0)

### PR DESCRIPTION
**Pull request by hal-shu-sato**
## Description

- Change to make an installation directory just before installing.

## Related Issue

Fix #388

## How Has This Been Tested?

- apm started normally.
- apm installed AviUtl, Exedit, plugins, and scripts.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by yumetodo — DISMISSED** [review]
https://github.com/hal-shu-sato/apm/blob/b93caae5fad9d12081f633a9bc85f7074ec13c4c/src/core/core.js#L388

copySyncがパスが存在しないときにdirectryを作成することを確認しました。

File: src/core/core.js [hunk 1]
```diff
@@ -262,33 +261,35 @@ async function selectInstallationPath(input) {
 async function changeInstallationPath(instPath) {
   store.set('installationPath', instPath);
 
-  // update 1
-  await mod.downloadData();
-  const currentMod = await mod.getInfo();
+  if (fs.existsSync(instPath)) {
```
Comment (mitosagi) [inline src/core/core.js:264]: `%AppData%/Roming/AviUtl...`を消去した状態で以下の処理を回避すると、AviUtl本体インストール用のデータを取得できなくなるようです。

**Comment by hal-shu-sato** [comment]
I fixed it.

**Review by mitosagi — APPROVED** [review]
It works fine.

**Pull request merged**